edi_oca/models/edi_exchange_record.py

```diff
@@ -259,6 +286,10 @@ def name_get(self):
     def create(self, vals_list):
         for vals in vals_list:
             vals["identifier"] = self._get_identifier()
+            if vals.get("res_id") and vals.get("model"):
```

Comment:
what about an inverse method on res_id?

Reply:
yes, the inverse is cleaner. However, it does not work correctly in the 'create' unless the res_id and model are readonly=False (I have also tried with the force_save on the view).

Another option would be to leave them readonly=False and making them readonly in the views

Reply:
I'd go for the last option